deploy/charts/istio-operator/Chart.yaml

```diff
@@ -9,7 +9,7 @@ sources:
 icon: https://istio.io/latest/img/istio-whitelogo-bluebackground-framed.svg
 
 # Based on support status of Istio releases: https://istio.io/latest/docs/releases/supported-releases/#support-status-of-istio-releases
-kubeVersion: ">= 1.20.0-0 <= 1.23.0-0"
+kubeVersion: ">= 1.21.0-0 <= 1.24.3-0"
 
-version: 2.0.12
-appVersion: "v2.13.4"
+version: 2.0.13
+appVersion: "v2.14.0"
```

Comment:
We usually release rc releases first for testing, but I'm not consisting if you can test otherwise as well easily, let's just not release the stable version until the code is reviewed/solution is tested.